Makefile

```diff
@@ -1,6 +1,10 @@
 # Install to /usr/local unless otherwise specified, such as `make PREFIX=/app`
 PREFIX?=/usr/local
 
+# Man pages by default instaled to /usr/share/man, like above this can
+# be specified on command line.
+MANPATH?=/usr/share/man
```

Comment:
```suggestion
# Man pages by default instaled to /usr/local/share/man, like above this can
# be specified on command line.
MANPATH?=$(PREFIX)/share/man
```

Reply:
Yeah honestly I was thinking of doing this earlier, but wasn't sure if `/usr/local/share/man` was a valid man path